ckanext/data_qld/helpers.py

```diff
@@ -21,6 +21,9 @@ def get_user():
 
 
 def user_has_admin_access(include_editor_access):
```

Comment:
time to rename this function. Its user_has_admin_access_or_background_task

Reply:
I'm not so sure it needs a rename...background tasks are part of the system, therefore they do get admin access.